__tests__/boot.ts

```diff
@@ -91,6 +91,7 @@ const LOGGED_IN_BODY = {
     company: null,
     cover: null,
   },
+  marketingCta: null,
```

Comment:
Could be seperate PR, but maybe let's add a test for testing that it adds one full object as well?

Reply:
It'll come in the next PR. Just added it to "reserve" the name for it. So I also had to add it to the tests to satisfy the tests 😬